test/Npgsql.Tests/Types/JsonTests.cs

```diff
@@ -61,6 +62,55 @@ public void RoundtripLongString()
             }
         }
 
+        [Test]
+        public void ReadJsonDocument()
+        {
+            using (var conn = OpenConnection())
+            using (var cmd = new NpgsqlCommand("SELECT @p", conn))
```

Comment:
We are using C# 8.0 and there is no need in curly braces in case of `using`s too.

Reply:
I agree, but let's do this in a big commit that does all the tests, OK?

Reply:
Okay, I will do it.